### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by spencersolomon6**
#5 - Adds frontend component and service support for displaying another user's profile to the currently authenticated user.

### CODE REVIEW

**Review by tmudunuri — APPROVED** [review]

File: src/components/Tuits/Tuit.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import TuitImage from './TuitImage';
 import TuitVideo from './TuitVideo';
```
Comment (sabrasisler) [inline src/components/Tuits/Tuit.js:4]: Can you make sure to add documentation at the top each file you edit or create?

File: src/services/users-service.js [hunk 2]
```diff
@@ -1,9 +1,8 @@
 import axios from 'axios';
```
Comment (sabrasisler) [inline src/services/users-service.js:1]: Can you make sure to add documentation at the top each file you edit or create?

File: src/views/OtherUserProfileView/ProfileNav.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import { NavLink } from 'react-router-dom';
```
Comment (sabrasisler) [inline src/views/OtherUserProfileView/ProfileNav.js:1]: Can you make sure to add documentation at the top each file you edit or create?

File: src/views/OtherUserProfileView/UsersDislikes.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
```
Comment (sabrasisler) [inline src/views/OtherUserProfileView/UsersDislikes.js:1]: Can you make sure to add documentation at the top each file you edit or create?

**Review by sabrasisler — CHANGES_REQUESTED** [review]
Everything looks good except the documentation, which I believe should be done before we merge.

**Review by sabrasisler — APPROVED** [review]
Thanks for updating the documentation. Everything looks good now and should be ready to merge. Diego needs to approve it first though

**Pull request merged**